tools/data-handler/src/calculate.ts

```diff
@@ -331,21 +231,22 @@ export class Calculate {
       this.project.paths.calculationFolder,
       Calculate.modulesFileName,
     );
-    let modulesContent: string = '';
     // Collect all available calculations
     const calculations = await this.project.calculations(ResourcesFrom.all);
 
+    const builder = new ClingoProgramBuilder();
```

Comment:
`ClingoProgramBuilder` could be a private member of `calculate.ts`

Reply:
I don't think that makes sense, because then we'd have to create a reset method(so we can reuse it) and then the code won't be able to run asyncronously. We do use mutexes, but I don't think we should depend on them for this case